### PR TITLE
Contentupdate

### PR DESCRIPTION
**Pull request by sccarmichael**
Done: 

- [x] Restructured IA
- [x] Updated all copy
- [x] Removed `Guides` link from the top nav

Remaining: 
- [ ] Create `About` or `Overview` page in top nav 
- [x] Rebuild links and imagery where appropriate
- [ ] Styling

### CODE REVIEW

File: docs/quick-start/handoff.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: handoff
+title: 4. Handoff
```
Comment (sbauch) [inline docs/quick-start/handoff.md:3]: there's another property here `sidebar_label` that you could use

**Comment by sbauch** [comment]
"Learn More" link in hero is broken

File: docs/quick-start/handoff.md (hunk 1 above)
Comment (sccarmichael) [reply docs/quick-start/handoff.md:3]: oh yeah, nice - forgot about that property. helpful!

File: sidebars.js [hunk 2]
```diff
@@ -1,41 +1,35 @@
...
-            "idp/azure-active-directory",
-            "idp/google-saml",
-            "idp/okta",
-            "idp/one-login",
-          ],
-        },
+        "consume/overview",
+        "consume/adding-oauth-clients",
+        "consume/consuming-osso",
+        "consume/redirect-uris",
```
Comment (sbauch) [inline sidebars.js:22]: can we swap the order here?

```suggestion
        "consume/redirect-uris",
        "consume/consuming-osso",
```

File: sidebars.js [hunk 3]
```diff
@@ -1,41 +1,35 @@
...
+      ],
+    },
+    {
+      type: "category",
+      label: "How to use Osso",
+      items: [
+        "user-guide/overview",
+        "user-guide/adding-customers",
+        "user-guide/onboarding-customers",
```
Comment (sbauch) [inline sidebars.js:31]: these feel a little redundant, but we can address later

**Review by sbauch — APPROVED** [review]
nice! one comment id like to address, otherwise 🚢

File: sidebars.js (hunk 3 above)
Comment (sccarmichael) [reply sidebars.js:31]: yeah fair point. one is more step-by-step, but there's a good amount of overlap

**Pull request merged**